lib/streamlit/dataframe_util.py

```diff
@@ -708,7 +708,7 @@ def convert_anything_to_sequence(obj: OptionSequence[V_co]) -> Sequence[V_co]:
         return []  # type: ignore
 
     if isinstance(
-        obj, (str, list, tuple, set, range, EnumMeta, deque, map)
+        obj, (str, list, tuple, set, range, EnumMeta, deque, map, ItemsView, enumerate)
```

Comment:
nit: could you maybe sort the tuple by some criterion (e.g. alphabetically) so that's easier also for humans to scan it 😅

Reply:
sorted it 👍